perlRun.launch

```diff
@@ -6,6 +6,6 @@
 </listAttribute>
 <stringAttribute key="org.eclipse.ui.externaltools.ATTR_LAUNCH_CONFIGURATION_BUILD_SCOPE" value="${none}"/>
 <stringAttribute key="org.eclipse.ui.externaltools.ATTR_LOCATION" value="${PERL}"/>
-<stringAttribute key="org.eclipse.ui.externaltools.ATTR_TOOL_ARGUMENTS" value="&quot;${resource_name}&quot; ${string_prompt:${resource_name} arguments:}"/>
+<stringAttribute key="org.eclipse.ui.externaltools.ATTR_TOOL_ARGUMENTS" value="&quot;${resource_name}&quot; ${string_prompt:${resource_name} arguments:-storename Store}"/>
```

Comment:
wouldn't like to integrate this... the perlRun launch is generic, should not default arguments to something that only applies to some scripts.